viewing_party/party.py

```diff
@@ -1,23 +1,181 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    movie_info = {}
+    if not title or not genre or not rating:
+        return None
+
+    # if title and genre and rating:
+
+    movie_info["title"] = title
+    movie_info["genre"] = genre
+    movie_info["rating"] = rating
+
+    return movie_info
+
+result = create_movie("", "Horror", "5")
+print(result)

```

Comment:
Print statements like these can be super useful when it comes to debugging, just remember to remove them from your final submission!